doc/ipythonconsole.rst

```diff
@@ -39,7 +39,9 @@ Connect to an external kernel
 
 To connect to an external kernel,
 
-#. Launch an IPython kernel on the local or remote host if one is not already running, e.g. with ``ipython kernel``.
+#. Launch an IPython kernel on the local or remote host if one is not already running.
+   If using Spyder 3.3.0 or later or Spyder 4, you'll want to do so with ``python -m spyder_kernels.console`` (after you've first installed ``spyder-kernels`` on the host with ``<conda/pip> install spyder-kernels``).
```

Comment:
Let's remove `Spyder 4` from here because it could confuse users since it's not released yet.

Reply:
Also, I'd prefer to change

> you'll want to do so ...

for

> you need to so ...

Reply:
Done and done.